### DISCUSSION TITLE
Anybody else experiences an annoying flicker on stacked/tabbed containers?

### DISCUSSION THREAD

**Discussion by edu-flores** [i3/i3#6192] [Q&A]
![glitch](https://github.com/user-attachments/assets/5e1bebcd-5c0e-421f-97f3-4ceb605c69a4)

After I switch workspaces, and then change tabs from a tabbed/stacked container, the previous workspace shows up for a split second. This happens both with `picom` enabled and disabled... I have a GTX 960 GPU. Any fixes?

**Comment by piotrnajda3000** [comment]
Ubuntu 22.04, GTX 1060 - same issue. Happens also with Full Composition Pipeline enabled.
